### PR TITLE
Feat/better messages

### PR DESCRIPTION
**Pull request by AlexanderHOtt**
There are a few things that changed in this PR

- All (important) message have been moved to `discord-bot/bot/messages.py`
- Messages look a lot better
- Users can only do 1 task at once

### CODE REVIEW

**Comment by yk** [comment]
LGTM so far, thank you very much!

@andreaskoepf I'll leave approval to you

**Comment by AlexanderHOtt** [comment]
this also fixes a bug where I was sending ranking responses as a `text` reply and not a `ranking` reply

File: discord-bot/bot/bot.py [hunk 1]
```diff
@@ -34,8 +34,11 @@ async def on_starting(event: hikari.StartingEvent):
 
     bot.d.oasst_api = OasstApiClient(settings.oasst_api_url, settings.oasst_api_key)
 
-    # A set of user id's that are currently doing work.
-    bot.d.currently_working = set()
+    # A `dict[hikari.Message | None, UUID | None]]` that maps user IDs to (task msg ID, task UUIDs).
+    # Either both are `None` or both are not `None`.
+    # If both are `None`, the user is not currently selecting a task.
+    # TODO: Grow this on startup so we don't have to re-allocate memory every time it needs to grow
```
Comment (andreaskoepf) [inline discord-bot/bot/bot.py:40]: This is in general not necessary (premature optimization).

File: discord-bot/bot/extensions/guild_settings.py [hunk 2]
```diff
@@ -78,7 +78,6 @@ async def log_channel(ctx: lightbulb.SlashContext) -> None:
 
     # if the bot's permissions for this channel don't contain SEND_MESSAGE
     # This will also filter out categories and voice channels
-    print(permissions_in(ch, own_member) & hikari.Permissions.SEND_MESSAGES)
```
Comment (andreaskoepf) [inline discord-bot/bot/extensions/guild_settings.py:81]: (you can use logger.debug() to add some debug-tracing ... sometimes there location where you inserted print statements for dev are actually neuralgic points and tracing them might be useful for other debugging too)

**Review by andreaskoepf — APPROVED** [review]
thnx! will test interaction after merge.

File: discord-bot/bot/bot.py (hunk 1 above)
Comment (AlexanderHOtt) [reply discord-bot/bot/bot.py:40]: I agree, it's just a reminder for the future. Should I change "TODO" to some other word?

File: discord-bot/bot/extensions/guild_settings.py (hunk 2 above)
Comment (AlexanderHOtt) [reply discord-bot/bot/extensions/guild_settings.py:81]: that was left from an upstream mistake, it will be corrected when this gets merged.

**Pull request merged**